src/previewSupport/handleConsoleLogs.js

```diff
@@ -0,0 +1,68 @@
+import isString from 'lodash/isString';
+import channel from './channel';
+// eslint-disable-next-line no-eval
+const globalEval = window.eval;
+
+function reducer(accumulator, currentValue) {
+  let evaluatedValue;
+  if (isString(currentValue)) {
+    evaluatedValue = currentValue;
+  } else {
+    evaluatedValue = globalEval(currentValue);
+  }
+
+  return `${accumulator} ${JSON.stringify(evaluatedValue)}`;
+}
+
+function notifyChannel(args) {
+  if (args.length > 0) {
+    const output = args.reduce(reducer);
```

Comment:
Isn’t this essentially a `map` + `join`? That seems like a simpler way to write it than a `reduce`.

Reply:
done